hashes/src/siphash24.rs

```diff
@@ -211,12 +210,57 @@ impl Hash {
     }
 
     /// Returns the (little endian) 64-bit integer representation of the hash value.
-    pub fn as_u64(&self) -> u64 { u64::from_le_bytes(self.0) }
+    pub fn as_u64(&self) -> u64 {
+        u64::from_le_bytes(self.0)
+    }
```

Comment:
Looks like you've accidentally formatted the code? IIRC we use nightly to format.

Also OT: this should be `to_u64`.

Reply:
#3114